compact/far_forward/B0_tracker.xml

```diff
@@ -164,7 +164,7 @@
 
   <readouts>
     <readout name="B0TrackerHits">
-      <segmentation type="CartesianGridXZ" grid_size_x="0.020*mm" grid_size_z="0.020*mm" />
+      <segmentation type="CartesianGridXZ" grid_size_x="0.070*mm" grid_size_z="0.070*mm" />
```

Comment:
```suggestion
      <comment>
        20um resolution = 75um "pixels", but this is "effective", since we assume charge sharing in the real-life 500um AC-LGADs - this may need to addressed differently in simulations later via digitization
      <comment>
      <segmentation type="CartesianGridXZ" grid_size_x="0.070*mm" grid_size_z="0.070*mm" />
```

Reply:
Good call!